Tests/TuistGeneratorTests/GeneratorTests.swift

```diff
@@ -1,35 +1,152 @@
 import Basic
 import XCTest
+@testable import TuistCoreTesting
```

Comment:
Similarly this is a move (we had a placeholder test file previously hence why it's highlighting some changes)